src/js/component.js

```diff
@@ -220,28 +220,35 @@ class Component {
    * @return {string}
    *         The localized string or if no localization exists the english string.
    */
-  localize(string) {
+  localize(string, tokens, defaultValue = string) {
```

Comment:
So that the key in the lang file describes what it should be. If you see `{1} of {2}` how would you know what is `{1}` and `{2}` and also what should you be translating?